spec/lib/cts/cases/cli_spec.rb

```diff
@@ -28,7 +29,12 @@
         size: 2
       }
     )
-    find_or_create :team_dacu
+    allow(cli_warehouse).to receive(:options).and_return(
+      {
+        size: 2
+      }
```

Comment:
Tiny thing but do we need all the vertical space here? 

```suggestion
      { size: 2 }
```